lib/src/main/java/com/orange/ods/compose/component/button/OdsButtonsCommon.kt

```diff
@@ -32,3 +35,11 @@ internal fun ButtonIcon(painter: Painter) {
     )
     Spacer(Modifier.size(ButtonDefaults.IconSpacing))
 }
+
+@Composable
+internal fun iconButtonIconColor(displaySurface: OdsDisplaySurface) =
```

Comment:
This was not modified in this PR but repeating `icon` twice in the name sounds a bit strange (or maybe it's me 😄).

What do you think of `iconButtonTintColor` ?